server/templates/sustainability/base.html

```diff
@@ -0,0 +1,188 @@
+  {% endblock %}
+</head>
+
+<body>
+  <div id="main">
+    <header id="main-header">
+      <nav class="navbar navbar-dark navbar-expand-lg col" id="main-nav">
+        <div class="container-fluid">
+          <div class="navbar-brand">
+            <a href="{{ url_for('static.homepage') }}">Sustainability Data Commons</a>
+            {% if subpage_title %}
+            <span>
+              {% if subpage_url %}<a href="{{ subpage_url }}">{% endif %}
+                {{ subpage_title }}
+                {% if subpage_url %}</a>{% endif %}
+            </span>
+            {% endif %}
+            </a>
+          </div>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#dc-main-nav">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse justify-content-md-end" id="dc-main-nav">
+            <ul class="navbar-nav float-md-right">
+              <li class="nav-item">
+                <a class="nav-link" href="https://datacommons.org">Data Commons</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="nav-explore-dropdown" role="button"
+                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  {% trans %}Explore{% endtrans %}
+                </a>
+                <div class="dropdown-menu" aria-labelledby="nav-explore-dropdown">
+                  <a class="dropdown-item" href="{{ url_for('place.place') }}">{% trans %}Place Explorer{% endtrans %}</a>
+                  <a class="dropdown-item" href="{{ url_for('tools.timeline') }}">{% trans %}Timelines Explorer{% endtrans %}</a>
+                  <a class="dropdown-item" href="{{ url_for('tools.scatter') }}">{% trans %}Scatter Plot Explorer{% endtrans %}</a>
+                  <a class="dropdown-item" href="{{ url_for('tools.map') }}">{% trans %}Map Explorer{% endtrans %}</a>
+                </div>
+              </li>
+            </ul>
+            {%- if not hide_search -%}
+            <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
+            {%- endif -%}
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main id="{{ main_id }}" class="main-content">
+      {% block content %}
+      {% endblock %}
+    </main>
+
+    <footer id="main-footer">
+      <div class="container">
+        {% if not is_hide_full_footer %}
+        <div class="row">
+          <section class="col-12 col-sm-6 col-md-4">
+            {# TRANSLATORS: The label for a collection of exploration tools. #}
+            <h6>{% trans %}Explore{% endtrans %}</h6>
+            {# TRANSLATORS: The name of a tool to browse statistics about a place. #}
+            <a href="{{ url_for('place.place') }}">{% trans %}Place Explorer{% endtrans %}</a>
+            {# TRANSLATORS: The name of a tool to browse the Data Commons knowledge graph. #}
```

Comment:
Are these translation comments needed here?

Reply:
thanks - not needed. removed.